.github/workflows/reusable-python-publish_package.yml

```diff
@@ -50,126 +50,95 @@ on:
         EXTRA_PYPI_REPO_TOKEN:
             description: 'Extra PyPI repository token. Only used if `extra-repository` is True.'
             required: false
-        MIRA_REPO_SECRETS:
-            description: 'Base64 encoded Yanml file content, containing all the repository secrets'
+        JFROG_ARTIFACTORY_URL:
+            description: 'JFrog Artifactory URL'
+            required: true
+        JFROG_ARTIFACTORY_TOKEN:
+            description: 'JFrog Artifactory Token'
             required: true
 
 defaults:
     run:
         shell: 'bash -l {0}'
 
 jobs:
-    release_pypi_package:
-        name: Release PyPI package
+    build_package:
+        name: Build package
+        strategy:
+            fail-fast: false
+            matrix:
+                package-manager: ${{ fromJSON(inputs.package-managers) }}
         runs-on: ubuntu-latest
+        defaults:
+            run:
+                shell: 'bash -l {0}'
         timeout-minutes: ${{ inputs.timeout-minutes }}
-        if: ${{ inputs.package-manager != 'conda' }}
-        env:
-            cache-number: 1
         steps:
-            -   uses: actions/checkout@v4
+            -   name: Checkout
+                uses: actions/checkout@v4
                 with:
                     lfs: ${{ inputs.lfs }}
                     fetch-depth: 0
-            -   name: Set up Python version
-                uses: actions/setup-python@v5
+            -   name: Get version
```

Comment:
**question:** is this step only required for conda?

I did not see the output version used for poetry

Reply:
The output version is used to order correctly our packages in JFrog Artifactory. This output is used in `Publish package to Artifactory` job:
`jf rt u $file ${repo_name}/${{ inputs.package-name }}/${{ steps.get-version.outputs.version }}/${filename}`